### DISCUSSION TITLE
Kubernetes metrics collection

### DISCUSSION THREAD

**Discussion by padmohan-clgx** [open-telemetry/opentelemetry-collector#9977] [Q&A]
Can we use centralized opentelemetry collector to collect Kubernetes Pod metrics ?

**Comment by TylerHelmuth** [comment]
Yup, checkout https://opentelemetry.io/docs/kubernetes/getting-started/

**Reply by SimranCode** [reply]
How and where do we specify the centralized remote OTEL Collector endpoint ?
Can someone please provide a sample or a  reference .

For context : 
I got a OTEL Collector running remotely , I want to send all the Kubernetes Infra metrics and logs to that endpoint and which internally will export it to Prometheus .
